aviary/interface/methods_for_level2.py

```diff
@@ -1259,26 +1296,76 @@ def add_post_mission_systems(self, include_landing=True):
                 ],
                 promotes_outputs=[('overall_fuel', Mission.Summary.TOTAL_FUEL_MASS)])
 
-            if 'constrain_range' in self.post_mission_info:
-                if self.post_mission_info['constrain_range']:
-                    target_range = self.post_mission_info['target_range']
+            regular_phases, reserve_phases = self.phase_separator()
```

Comment:
So, i think the mass residual is a little tricky.  If you look at line 1287, you find the execcomp with the equation `overall_fuel = fuel_burned + fuel_reserve`.  The way things are done now, the `fuel_burned = starting mass - mass at end of final phase`. This means that, with the additional reserve phases, reserve mission is considered part of fuel_burned (start of climb - end of final reserve).  The `fuel_reserve` variable here is Mission.Design.RESERVE_FUEL, which is a user-specified amount in lbs, and this might be 0 for a model like this because we have a reserve mission instead.

Maybe it is semantics, but including the reserve fuel in the fuel burned is going to confuse users, who are expecting that variable to be just them mission fuel. A solution to this requires separately computing the fuel burned in the normal and the reserve phases, and adding it into this equation.  (Side note, i recommend adding it as another var, because Jason has proposed that users might want to run a reserve mission AND specify an additional reserve fuel in lbs.)

Reply:
we could use block_fuel for fuel_burned and reserve_mission_fuel for the fuel used during reserve phases (separate from the fixed reserve_fuel)

Reply:
I looked through the variable metadata, apparently there are three reserve fuel names:
RESERVE_FUEL_ADDITIONAL, RESERVE_FUEL_FRACTION, RESERVE_FUEL

Reply:
RESERVE_FUEL_ADDITIONAL is a fixed amount (e.g. 5000 lbs)
RESERVE_FUEL_FRACTION is a percentage of the design mission fuel (e.g. 5%)
RESERVE_FUEL is the total (currently of just additional and fractional, but should include reserve_mission_fuel when it is available)

Reply:
Mission.Summary.FUEL_BURNED and Mission.Summary.RESERVE_FUEL_BURNED have been added to variable hierarchy and to methods for lvl2. Fuel-flow calcs have been re-done to reflect this.